### ISSUE TITLE
Plan changing assertion verdicts to PASS/FAIL from GOOD/NO/INCORRECT output

### ISSUE THREAD

**Issue by jugglinmike** [w3c/aria-at#961]
Currently, assertion verdicts can have one of three possible values. The words used to describe those values are contextual:

Working Mode term     | ARIA-AT App term
----------------------|-----------------
supported             | good output
not supported         | no output
incorrectly supported | incorrect output

The community group agreed to reduce the number of states to just two: "pass" and "fail." See w3c/aria-at#945 for the discussions that motivated this change.

#### Implementation Requirements

This issue is complete when the following issues are fully specified and planned.

- [ ] w3c/aria-at#969
- [x] w3c/aria-at-app#735
- [ ] w3c/aria-at-app#736

**Comment by ccanash** [comment]
Closing as done as the associated tasks are already in production. cc @mcking65
